### PR TITLE
Improved 30 minute warning text on ticket page.

### LINKED ISSUE #132: The 30 minute warning sounds a little harsh

### PR DESCRIPTION
**Pull request by bvanderdrift**
Fixes #132

### CODE REVIEW

File: app/elements/lancie-ticket-page/lancie-ticket-page.html [hunk 1]
```diff
@@ -58,7 +58,7 @@
                   <paper-button raised on-tap="openDialog" disabled="{{hasNoTickets}}">Check out</paper-button>
                   <br />
                   <br />
-                  <small>Note: If you do not pay the tickets within 30 minutes, the order will be removed.</small>
+                  <small>Note: Upon pressing the button the tickets will be reserved and you will have 30 minutes to pay. Tickets will be released after this period if not paid.</small>
```
Comment (skamoen) [inline app/elements/lancie-ticket-page/lancie-ticket-page.html]: "The button". Which button? I don't think this is clear enough.

**Comment by martijn9612** [comment]
image
The 2 containers in the ticket page are overlapping and not properly aligned.

**Comment by TimvdLippe** [comment]
You mean you want a margin between the cards? They are strictly not overlapping, but touching.

And with aligned I suppose you mean the height of the card? This would need a min-height, as adding tickets will expand this card instead.

**Comment by martijn9612** [comment]
I was actually not protesting the height of the card, it was mainly about the spacing between them. The non-overlap but touching just looks very odd in my opinion. Every page has spacing between the cards except here. If you narrow the screen and the cards are on top of each other, the spacing is present (although top to bottom). Spacing just makes the design look a lot cleaner and more unified across the entire website.

**Comment by TimvdLippe** [comment]
Ready again for review.

**Comment by skamoen** [comment]
It's not displaying the weird overlap behaviour anymore. There's one nitpicky detail left though:

image

The two paper-materials aren't the same width

**Comment by skamoen** [comment]
LGTM

**Pull request merged**